mongodb_communityserver/test.sikuli/test.py

```diff
@@ -0,0 +1,58 @@
+script_path = os.path.dirname(os.path.abspath(sys.argv[0])) 
```

Comment:
If the tests are identical, maybe we can add a comment at the top.

Reply:
added comment in new commit